lib/helper.js

```diff
@@ -1,18 +1,21 @@
 'use strict';
 
 const runScript = require('runscript');
-const REGEX = /^\s*(\d+)\s+(.*)/;
+const isWin = process.platform === 'win32';
+const REGEX = isWin ? /^(.*)\s+(\d+)\s+$/ : /^\s*(\d+)\s+(.*)/;
 
 exports.findNodeProcess = function* (filterFn) {
-  const command = 'ps -eo "pid,command"';
+  const command = isWin ?
+    'wmic Path win32_process Where "Name = \'node.exe\'" Get CommandLine,ProcessId' :
+    'ps -eo "pid,command"';
   const stdio = yield runScript(command, { stdio: 'pipe' });
   const processList = stdio.stdout.toString().split('\n')
     .reduce((arr, line) => {
       if (!!line && !line.includes('/bin/sh') && line.includes('node')) {
         const m = line.match(REGEX);
         /* istanbul ignore else */
         if (m) {
-          const item = { pid: m[1], cmd: m[2] };
+          const item = isWin ? { pid: m[2], cmd: m[1] } : { pid: m[1], cmd: m[2] };
```

Comment:
why not just change  `Get CommandLine,ProcessId` to `Get ProcessId,CommandLine` ?

Reply:
+1 after fix this

Reply:
In my test, `ProcessId` always show behind `CommandLine`, despite how you run the command.

So I just put `ProcessId` behind `CommandLine`, more clear, less confuse.

Reply:
how about change the `ps` of linux to make they the same order ?

Reply:
It's still confusing, `Get CommandLine,ProcessId`  and `Get ProcessId,CommandLine` actually output the same thing, maybe I should add a comment in the code ? Got better idea? @atian25

Reply:
`ps -eo "pid,command"` -> `ps -eo "command,pid"`

Reply:
That's better, hahaha

Reply:
something wrong with the `REGEXP` in my final commit, I will switch to linux to test it.

Reply:
It's weird:

`ps -eo pid,command` works fine

```
fake@ubuntu$ ps -eo pid,command
  PID COMMAND
    1 /init ro
   24 ssh-agent
  205 /init ro
  206 -bash
 2956 /init ro
 2957 -bash
 5274 npm
 5284 sh -c npm run lint -- --fix && npm run pkgfiles && npm run test-local
 5325 npm
 5335 sh -c egg-bin test
 5336 node /mnt/c/Users/fake/code/egg-scripts/node_modules/.bin/egg-bin test
 5342 /usr/local/lib/nodejs/node/bin/node /mnt/c/Users/fake/code/egg-scripts/node_modules/egg-bin/node_modules/mocha/bin/_mocha --require=/mnt/c/Users/fake/code/egg-scripts/node_modules/egg-bin/lib/mocha-clean.js --require=/mnt/c/U...
 5595 node /mnt/c/Users/fake/code/egg-scripts/lib/start-cluster {"framework":"/mnt/c/Users/fake/code/egg-scripts/test/fixtures/example/node_modules/yadan","workers":2,"title":"egg-server-example","baseDir":"/mnt/c/Users/fake/cod...
 5680 /usr/local/lib/nodejs/node/bin/node /mnt/c/Users/fake/code/egg-scripts/bin/egg-scripts.js start --workers=2 --title=egg-test /mnt/c/Users/fake/code/egg-scripts/test/fixtures/example
 5690 node /mnt/c/Users/fake/code/egg-scripts/lib/start-cluster {"workers":2,"title":"egg-test","framework":"/mnt/c/Users/fake/code/egg-scripts/test/fixtures/example/node_modules/custom-framework","baseDir":"/mnt/c/Users/fake/co...
 5696 /usr/local/lib/nodejs/node/bin/node /mnt/c/Users/fake/code/egg-scripts/node_modules/egg-cluster/lib/agent_worker.js {"framework":"/mnt/c/Users/fake/code/egg-scripts/test/fixtures/example/node_modules/custom-framework","baseDi...
 5706 /usr/local/lib/nodejs/node/bin/node /mnt/c/Users/fake/code/egg-scripts/node_modules/egg-cluster/lib/app_worker.js {"framework":"/mnt/c/Users/fake/code/egg-scripts/test/fixtures/example/node_modules/custom-framework","baseDir"...
 5707 /usr/local/lib/nodejs/node/bin/node /mnt/c/Users/fake/code/egg-scripts/node_modules/egg-cluster/lib/app_worker.js {"framework":"/mnt/c/Users/fake/code/egg-scripts/test/fixtures/example/node_modules/custom-framework","baseDir"...
```

`ps -eo command,pid` faild: `command` was truncated

```
fake@ubuntu$ ps -eo command,pid
COMMAND                       PID
/init ro                        1
ssh-agent                      24
/init ro                      205
-bash                         206
/init ro                     2956
-bash                        2957
npm                          5274
sh -c npm run lint -- --fix  5284
npm                          5325
sh -c egg-bin test           5335
node /mnt/c/Users/fake/c     5336
/usr/local/lib/nodejs/node/  5342
node /mnt/c/Users/fake/c     5595
/usr/local/lib/nodejs/node/  5680
node /mnt/c/Users/fake/c     5690
/usr/local/lib/nodejs/node/  5696
/usr/local/lib/nodejs/node/  5706
/usr/local/lib/nodejs/node/  5707
ps -eo command,pid           5741
```

refer to [how-to-prevent-ps-from-truncating-the-process-name](https://askubuntu.com/questions/717919/how-to-prevent-ps-from-truncating-the-process-name), I think I should rollback :) ? or set column width like: `ps -eo command:1000,pid` ?

I'd like to rollback, any idea?

Reply:
@atian25

Reply:
no, just rollback to last review, thanks

Reply:
@atian25 done. Test on [appveyor](https://ci.appveyor.com/project/BaffinLee/egg-scripts) passed too.